app/javascript/components/widgets/feed/panel.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { colors, fontSizes, fonts, spacing } from '@lib/theme';
+import StarIcon from '@assets/images/icons/icon-star.svg';
+import GiftIcon from '@assets/images/icons/icon-gift.svg';
+import UsersIcon from '@assets/images/icons/icon-users.svg';
+import UserPlusIcon from '@assets/images/icons/icon-user-plus.svg';
+import { Row } from '@components/row';
+import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+import {useQuery} from '@apollo/client';
+import { LoadingMessage, DisconnectedMessage } from '@messages/message';
+import { takeLast, splitAt } from 'ramda';
+import { Carousel } from 'react-responsive-carousel';
+import 'react-responsive-carousel/lib/styles/carousel.css';
+
+const Title = styled.div`
+  margin-bottom: 50px;
+  margin-left: 50px;
+  width: 45vw;
+`;
+
+const imgStyle = {
+  margin: "auto",
+  top: "50%",
+  left: "50%",
+  objectFit: "cover",
+  display: "block"
+}
+
+const getMojoFeeds = gql`
+  query getFeeds {
+    feeds {
+      source
+      author
+      uuid
+    }
+  }
+`;
+
+const getMojoDoggos = gql`
+query getDoggos {
+  feeds (source: "C03T327NG68"){
+    uuid
+  }
+}
+`;
+
+const PanelWrapper = styled.div`
+  overflow: hidden;
+  height: 90%;
+`;
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
+
+
+function Feed({ feeds }) { 
+  
+  return (
+    <PanelWrapper>
+      <Title>
+        <WhiteTitleLarge>Images at MojoTech</WhiteTitleLarge>
+        <WhiteSubTitle>#mojodoggos</WhiteSubTitle>
+      </Title>
+      <Carousel
+        width="60%"
+        autoPlay={true}
+        showThumbs={false}
+        interval={3000}
+        showStatus={false}
+      >
+        <div>
```

Comment:
This could be re-factored & iterated over a range from 0..5.

Reply:
👍 why not just display them all? Let the backend worry about which ones to include?
```jsx
feeds.map(({uuid})=> <img src ={urlFromId(id) />})
```